Sources/WebDriver/Session.swift

```diff
@@ -73,6 +73,10 @@ public class Session {
             Requests.SessionTimeouts(session: id, type: type, ms: duration * 1000))
     }
 
+    public func execute(javascript script: String,  args: [String], async: Bool) throws {
```

Comment:
Better stick with the param name from the spec, and we can make the args optional
```suggestion
    public func execute(script: String, args: [String] = [], async: Bool) throws {
```